src/components/Search.js

```diff
@@ -56,20 +57,13 @@ export const Search = () => {
     })
   }
 
-  const goToRandomArticle = () => {
-    const [promise] = getRandomArticleTitle(lang)
-
-    promise.then(title => {
-      goto.article(lang, title)
-    })
-  }
-
   useSoftkey('Search', {
     right: i18n('softkey-settings'),
     onKeyRight: () => { window.location.hash = '/settings' },
     center: current.type === 'DIV' ? i18n('centerkey-select') : '',
     onKeyCenter,
-    onKeyLeft: isRandomEnabled() ? goToRandomArticle : null,
+    left: i18n('softkey-tips'),
+    onKeyLeft: () => { window.location.hash = '/tips' },
```

Comment:
This should be something like `goto.tips()`, which should be something like `route('/tips')`

Reply:
Adding a `goto.settings()` as well